book/website/community-handbook/coworking.md

```diff
@@ -0,0 +1,26 @@
+# Community Coworking Calls
+
+The concept of coworking is generally associated with the practice of sharing physical office space with different people who work independently such as the remote employees or freelancers.
+This allows people, who otherwise may feel isolated while working alone at home, to use traditional office setups, create a work routine, and build a sense of companionship with others who they share their office spaces with.
+
```

Comment:
```suggestion
The concept of coworking is generally associated with the practice of sharing physical office space with different people who work independently such as remote employees or freelancers.
This allows people, who otherwise may feel isolated while working alone at home, to use traditional office setups, create a work routine, and build a sense of companionship with others who they share their office spaces with.

```